app/models/storage_location.rb

```diff
@@ -15,6 +17,13 @@
 class StorageLocation < ApplicationRecord
   require "csv"
```

Comment:
Is `csv' needed? I'd have thought we were including this at the application level somewhere. Maybe not :)

Reply:
@seanmarcia Well, this require was there already, so I didn't change anything or included anything about it

Reply:
@seanmarcia I took a look at this and there are other files that require it, so I don't think it was included at the application level

Reply:
Cool! I was just curious -- I haven't worked with storage locations much in the past.

Reply:
Would it worth to create an issue to move this requirement to a more generic location?

Reply:
@wenderjean The CSV exporting was originally refactored out into a separate model but there were some complications that were emerging (mostly regarding filtering) from having it be separate.

It might be possible to make this a concern that's included in these models, but the responsibility of casting the object to CSV format should be left to the model itself. 

I think you can remove the requirement at the top, though!